splitwise/config.py

```diff
@@ -0,0 +1,2 @@
+consumer_key = '99WIaVrXvVaJDHYKAtZzZgFy145lSAE507PJFypd'
```

Comment:
You don't need to share keys on public platform. 
```suggestion
consumer_key = ''
```

Reply:
Okay!